sdk/src/main/java/com/horizen/account/node/NodeAccountState.java

```diff
@@ -1,9 +1,8 @@
 package com.horizen.account.node;
 
-import com.horizen.account.state.Account;
 import com.horizen.account.state.AccountStateReader;
 import com.horizen.node.NodeStateBase;
 
 public interface NodeAccountState extends NodeStateBase, AccountStateReader {
```

Comment:
Do we still need NodeStateBase? Now both its derived traits NodeAccountState and NodeState are empty

Reply:
in future we may put into `NodeStateBase` something common to both UTXO and Account model, for example withdrawal epoch info.
At the moment we keep it empty because there is no UTXO state implemented compatible to the new model